recipes/influxdb-python/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "influxdb-python" %}
+{% set version = "5.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/i/influxdb/influxdb-{{ version }}.tar.gz
+  sha256: afeff28953a91b4ea1aebf9b5b8258a4488d0e49e2471db15ea43fd2c8533143
+
+build:
+  number: 0
```

This looks like it could be a [noarch package](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-builds):
```suggestion
  number: 0
  noarch: python
```